featuretools/entityset/entityset.py

```diff
@@ -975,25 +982,28 @@ def add_last_time_indexes(self, updated_dataframes=None):
                     # so we didn't need this logic
                     for df in child_dataframes:
                         if df.ww.name not in explored and df.ww.name not in [q.ww.name for q in queue]:
+                            # must also reset last time index here
+                            es_lti_dict[df.ww.name] = None
```

Comment:
This allows us to reset the children that don't get added to the queue until now (whereas the parents get added at the beginning, and we set the last time indexes to be None there)